DemoApp/Sources/Views/CallView/CallingView/SimpleCallingView.swift

```diff
@@ -161,6 +168,13 @@ struct SimpleCallingView: View {
                 }
         }
     }
+
+    private func setPreferredVideoCodec(for callId: String) async {
```

Comment:
nit: maybe we can have an extension method defined on the view, since I see this used in a couple of places.

Reply:
I'm not against it but my argument is that this will "pollute" the View's API for something that is being used only in 2 places in the Demo App.

Reply:
ok, fair enough